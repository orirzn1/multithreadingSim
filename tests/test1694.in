L15
S8
O7
N8

T0
I@0x00000000
ADD $4, $4, $6
LOAD $2, $4, 0x4
LOAD $4, $5, $2
SUBI $5, $5, 0x4
ADDI $1, $3, 0x4
ADDI $7, $2, 0x4
ADD $3, $5, $6
ADD $6, $5, $7
SUB $6, $0, $3
SUB $1, $6, $3
HALT $0

T1
I@0x00000000
SUB $0, $4, $6
SUBI $2, $4, 0x3
SUB $5, $4, $6
LOAD $0, $2, $3
ADDI $3, $3, 0x2
SUBI $0, $1, 0x1
ADDI $1, $0, 0x4
SUBI $1, $2, 0x3
ADDI $4, $4, 0x1
ADDI $6, $2, 0x1
HALT $0

T2
I@0x00000000
ADDI $3, $4, 0x1
ADD $6, $2, $6
SUBI $4, $4, 0x2
LOAD $2, $3, 0x4
ADD $3, $0, $1
HALT $0

T3
I@0x00000000
STORE $4, $6, $5
LOAD $6, $6, 0x2
SUB $6, $2, $0
ADD $4, $0, $5
SUB $1, $3, $0
ADD $6, $4, $7
STORE $5, $7, $3
LOAD $7, $6, 0x2
SUB $4, $7, $6
HALT $0

T4
I@0x00000000
ADD $2, $6, $6
LOAD $7, $1, $2
LOAD $4, $5, 0x3
ADD $6, $7, $5
ADD $4, $7, $1
STORE $7, $3, $1
ADDI $2, $5, 0x2
ADDI $6, $4, 0x4
HALT $0

T5
I@0x00000000
SUB $6, $0, $0
STORE $0, $7, 0x2
ADD $4, $3, $2
LOAD $0, $3, 0x1
SUB $5, $5, $7
HALT $0

T6
I@0x00000000
SUB $6, $7, $2
ADDI $7, $2, 0x1
SUB $5, $5, $4
ADD $3, $6, $4
ADD $1, $1, $7
ADDI $3, $2, 0x4
HALT $0

T7
I@0x00000000
SUBI $5, $0, 0x1
ADD $5, $4, $4
ADDI $7, $2, 0x3
STORE $7, $3, 0x3
HALT $0

D@0x0
0x2
0xA
0x5
0x5
0x4
0x9
0x8
0x3
