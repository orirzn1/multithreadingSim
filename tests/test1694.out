
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x8	R2 = 0xA	R3 = 0xFFFFFFFC	R4 = 0x5	R5 = 0xFFFFFFFC	R6 = 0x4	R7 = 0xE
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFA	R2 = 0xFFFFFFFD	R3 = 0x2	R4 = 0x1	R5 = 0x0	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0xA	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x2	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x4	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x3
Blocked MT CPI for this program 3.338462

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x8	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x0	R5 = 0xFFFFFFFC	R6 = 0x4	R7 = 0x4
Register file thread id 1:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFA	R2 = 0xFFFFFFFD	R3 = 0x2	R4 = 0x1	R5 = 0x0	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x2	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x4	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x3
Finegrained Multithreading CPI for this program 1.138462

